Automattic/jetpack
# Fix EOL for module-headings.php

When I was dealing with compatibility issues in https://github.com/Automattic/jetpack/pull/9905 I somehow missed the `File has mixed line endings; this may cause incorrect results` warning (PR wasn't rebased?). This PR simply converts the file to CRLF EOL format

#### Changes proposed in this Pull Request:

* Convert `module-headings.php` to CRLF EOL format.

#### Testing instructions:

run `yarn php:5.2-compatibility modules/module-headings.php` and make sure there is no warnings or errors
